app/components/trainees/filters/view.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class='moj-filter-layout abigail'>
+  <div class="moj-filter-layout__filter app-filter">
+    <div class="moj-filter">
+      <div class="moj-filter__header">
+        <div class="moj-filter__header-title">
+          <h2 class="govuk-heading-m">Filters</h2>
+        </div>
+      </div>
+
+      <div class="moj-filter__content">
+
+        <% if !active_filters.empty? %>
+        <div class="moj-filter__selected">
+          <div class="moj-filter__selected-heading">
+            <div class="moj-filter__heading-title">
+              <h2 class="govuk-heading-m">Selected filters</h2>
+            </div>
+            <div class="moj-filter__heading-action">
+              <p class="govuk-body">
+              <%= govuk_link_to 'Clear', trainees_path, class: 'govuk-link--no-visited-state' %>
+              </p>
+            </div>
+          </div>
+
+          <% active_filters.each do |filter, value| %>
+            <h3 class="govuk-heading-s govuk-!-margin-bottom-0"><%= filter.humanize %></h3>
+            <ul class="moj-filter-tags">
+              <% tags_for_active_filter(filter, value).each do |tag| %>
+                <li>
+                  <%= link_to(tag[:title], tag[:remove_link], class: "moj-filter__tag") %>
+                </li>
+              <% end %>
+            </ul>
+          <% end %>
+        </div>
+      <% end %>
+
+        <div class="moj-filter__options">
+
+          <form method="get">
```

Comment:
Are we not able to use the GOV Uk form builder here? As it'll generate most of this markup I think

Reply:
I don't believe so - I got a lot of the way there using the form builder, but couldn't then explicitly set the `checked` or `selected` value on the fields based on the active params.